repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/filters/valkyrie/CollectResourceBaseOnPermissionTest.groovy

```diff
@@ -277,7 +277,7 @@ class CollectResourceBaseOnPermissionTest extends ReposeValveTest {
 
         then: "check response"
         mc.handlings.size() == 1
-        mc.receivedResponse.code == "500"
+        mc.receivedResponse.code == "502"
```

Comment:
This is another one where we would have returned a `500` before, but now return a `502`.